src/migrations/1732280885581-create-test-data.ts

```diff
@@ -0,0 +1,173 @@
+        INSERT INTO public.association
+        (id, "name", logo, description, telegram, phone, email, membership_fee)
+        VALUES(8, 'Aalto Daigaku', 'https://www.ayy.fi/sites/g/files/flghsv231/files/styles/o_690w_ah_n/public/2019-05/aalto_daigaku_maajanbu_logo.png?itok=NvFw0elV', 'Aalto Daigaku Maajanbu ry (or アールト大学麻雀部) is an Espoo-based association founded in September 2013 to bring together players of Riichi mahjong and promote mahjong at Aalto University.
+        
+        Riichi mahjong is a japanese variation of the old Chinese 4-player game. This is not the solitaire mahjong which many may know, but a complex and exciting game popular in Japan.
+        
+        Our association is open for everyone interested in Riichi mahjong. Our activities include weekly mahjong sessions during semesters as well as tournaments and excursions.', '@aaltodaigaku', '+358 888 888', 'aaltodaigaku@email.com', 22);
+        INSERT INTO public.association
+        (id, "name", logo, description, telegram, phone, email, membership_fee)
+        VALUES(9, 'Otakut', 'https://www.ayy.fi/sites/g/files/flghsv231/files/styles/o_690w_ah_n/public/2024-07/Otakut-Logo-2013%20%281%29.png?itok=WW8Q-zwZ', 'Otakut is an anime club in Espoo, affiliated with Aalto University''s Student Union since May 1996. We support hobbies related to Japanese animated movies, series and comics.
+        
+        Activities
+        - Anime Evenings
+        - Movie Nights and Marathons
+        - Game and Karaoke Nights
+        - Friday Social Events
+        - Outings and Collaborations
+        - Convention Attendance
+        
+        Events are centered in Otaniemi.
+        ', '@otakut', '+358 888 888', 'otakut@email.com', 11);
+        INSERT INTO public.association
+        (id, "name", logo, description, telegram, phone, email, membership_fee)
+        VALUES(10, 'Aalto-Basket', 'https://www.ayy.fi/sites/g/files/flghsv231/files/styles/o_690w_ah_n/public/2019-05/aaltobasket_logo.png?itok=dGaniM1S', 'Aalto-Basket is the basketball team of Aalto University from Espoo.
+        Our club has long traditions of student sports, being founded way back in 1981.
+        During recent years, the club has grown, and nowadays consists of four teams:
+        
+        • Men’s 1st team: Plays in 2nd division, practices twice a week
+        • Men’s 2nd team: Plays in 4th division, practices twice a week
+        • Men’s 3rd team: Plays in 5th division, practices once a week
+        • Women’s team: Playes in 4th division, practices twice a week
+        
+        Each team is looking for new players for the upcoming season (or just to join our practices) – So, feel free to contact us!', '@aaltobasket', '+358 888 888', 'aaltobasket@email.com', 23);
+        
+
+        INSERT INTO public."event"
+        (id, title, description, picture, "date", "location", price, capacity, association_id)
```

Comment:
You are putting association_id by hand which works well in a script like this where you know all the IDs, but when you fix that you will not be able to rely on ids. For this reason you should get the association_id with a subselect that gets the id of the association with a specific name (SELECT id FROM associations ass WHERE ass.name = <desired name>)